HIP/hip-0000-free-network-info-query.md

```diff
@@ -0,0 +1,63 @@
+| hip               | title              | author         | type            | status         | created       | discussions |
+| ----------- | -----------  | ----------- | ----------- |----------- | ----------- | ----------- |
+| TBD            | Free network info query | Simi Hunjan (@SimiHunjan ) |Service| Draft | 2021-06-09   |   https://github.com/hashgraph/hedera-improvement-proposal/discussions/82
+
+## Abstract
+
+The HIP proposes a free address book query.
+
+## Motivation
+
+The address book files 0.0.101 and 0.0.102 collectively maintain information about the nodes in a given Hedera network (previewnet, testnet, mainnet). The two files contain the following [information](https://github.com/hashgraph/hedera-protobufs/blob/main/services/BasicTypes.proto#L356) for each node:
+
+- IP address
+- Port number
+- Node account ID
+- Node ID
+- Node certificate hash
+- RSA_PublicKey
+- Description
+- Stake
+
+Today, applications have to use the File Service and submit two file contents query requests to return all the required node information. This results in two transaction fees for processing each of the queries. The information contained within the address book files are required for clients to send Hedera transactions or queries to valid Hedera consensus nodes.
```

Comment:
`The information contained within the address book files **_is_** required`

Reply:
resolved